ocs_ci/deployment/hosted_cluster.py

```diff
@@ -1509,7 +1513,13 @@ def create_subscription(self):
         hosted_odf_version = (
             config.ENV_DATA.get("clusters").get(self.name).get("hosted_odf_version")
         )
-        if "latest" in hosted_odf_version:
+
+        if self.catalog_source_on_management_cluster:
+            subscription_data["spec"]["source"] = templating.load_yaml(
+                constants.SUBSCRIPTION_YAML
+            )["spec"]["source"]
+
+        if "latest-" in hosted_odf_version:
```

Comment:
I'm just curious about this change - what is the reason for adding the dash as suffix to the `latest`  here? There is some case where it will not work in the original code?

Reply:
"-" might be not necessary, it just makes validation of image string more strict